src/data.ts

```diff
@@ -83,7 +84,7 @@ export type DataSource = UrlData | InlineData | NamedData;
 
 export type Data = DataSource | Generator;
 
-export type InlineDataset = number[] | string[] | boolean[] | object[] | string | object;
+export type InlineDataset = number[] | string[] | boolean[] | Dict<unknown>[] | string | Dict<unknown>;
```

Comment:
the `object[]` syntax was working ok, now I need to `extend Dict<unknown>` ...

Reply:
Can you expand on this comment? I don’t understand the context.

Reply:
I ended up doing an inline cast to `Dist<any>` [here](https://github.com/microsoft/vegalite-for-powerbi/blob/93aebf4ec2f360f800d828e4a4432101fa040d9f/src/visual.ts#L98). Let's see if any other users start seeing compiler issues with the removal of `object`.